addons/account/tests/test_reconciliation.py

```diff
@@ -471,14 +471,21 @@ def _move_revert_test_pair(move, revert):
             move_lines = _determine_debit_credit_line(move)
             revert_lines = _determine_debit_credit_line(revert)
 
+            # In Storno, move_lines[i] will reconcile with revert_lines[i], we keep the same columns
+            # In non-Storno, move_lines[i] will reconcile with revert_lines[(i+1)%2]
+            # because we swap debit/credit columns
+            move_indices, revert_indices = (0, 1), (1, 0)
+            if move.is_storno and revert.is_storno:
+                move_indices, revert_indices = (0, 0), (1, 1)
```

Comment:
:scream: au secours

filter the lines from move_lines and revert_lines on the reconciliable account instead of this... craziness